include/shuffle/GatherShuffle.h

```diff
@@ -0,0 +1,32 @@
+#pragma once

```

Comment:
This belongs in the benchmarks folder to make it clear it is not an actual implementation

Reply:
Good idea, let me tidy that up.